DataManagementSystem/Utilities/CSHelpers.py

```diff
@@ -0,0 +1,235 @@
+"""
+  This module contains helper methods for accessing operational attributes or parameters of DMS objects
+
+"""
+
+from DIRAC.ConfigurationSystem.Client.Helpers.Path      import cfgPath
+from DIRAC.ConfigurationSystem.Client.Helpers.Operations import Operations
+from DIRAC import gConfig, gLogger, S_OK, S_ERROR
+from _abcoll import Mapping
+# from Core.Utilities import SitesDIRACGOCDBmapping
+
+LOCAL = 1
+PROTOCOL = 2
+DOWNLOAD = 3
+
+def resolveSEGroup( seGroupList ):
+  seList = []
+  if type( seGroupList ) != type( [] ):
+    seGroupList = [seGroupList]
+  for se in seGroupList:
+    seConfig = gConfig.getValue( '/Resources/StorageElementGroups/%s' % se, se )
+    if seConfig != se:
+      seList += [se.strip() for se in seConfig.split( ',' )]
+      # print seList
+    else:
+      seList.append( se )
+    res = gConfig.getSections( '/Resources/StorageElements' )
+    if not res['OK']:
+      gLogger.fatal( 'Error getting list of SEs from CS', res['Message'] )
+      return []
+    for se in seList:
+      if se not in res['Value']:
+        gLogger.fatal( '%s is not a valid SE' % se )
+        seList = []
+        break
+
+  return seList
+
+
+class CSHelpers():
+
+  def __init__( self ):
+    self.local = LOCAL
+    self.protocol = PROTOCOL
+    self.download = DOWNLOAD
+    self.siteSEMapping = {}
+    self.storageElementSet = set()
+    self.siteSet = set()
+    self.__opsHelper = Operations()
+
+
+  def getSiteSEMapping( self ):
```

Comment:
Yes, I know... The problem with these helpers is that they contain everything for all systems and their maintenance is rather dubious... 
In order to implement the long-discussed SE-Site relationship, I needed to implement it on the current v6 CS structure (while it was promised for v7 that doesn't come).
I know there are helpers here and there mostly as functions but this doesn't allow any caching. Therefore I propose an implementation with a class (the name can be changed as it is not reflecting the fact that it is DM-specific).